packages/cosmos_ui_components/lib/components/template/transaction_history_card.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:cosmos_ui_components/cosmos_text_theme.dart';
+import 'package:cosmos_ui_components/cosmos_ui_components.dart';
+import 'package:cosmos_ui_components/utils/date_formatter.dart';
+import 'package:flutter/material.dart';
+
+class TransactionHistoryCard extends StatelessWidget {
+  const TransactionHistoryCard({
+    Key? key,
+    required this.isOutgoing,
+    required this.transactionType,
+    required this.date,
+    required this.amountText,
+    required this.denomText,
+  }) : super(key: key);
+
+  final bool isOutgoing;
+  final String transactionType;
+  final DateTime date;
+  final String amountText;
+  final String denomText;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = CosmosTheme.of(context);
+    return Padding(
+      padding: EdgeInsets.symmetric(horizontal: theme.spacingL, vertical: theme.spacingM),
+      child: Column(
+        children: [
+          Row(
+            children: [
+              Image.asset(
+                isOutgoing ? 'assets/images/send_icon.png' : 'assets/images/receive_icon.png',
```

Comment:
we can optimize it and instead of having two separate assets for the same icon, we can simply add rotation to the image based on the `isOutgoing` bool value